chain/chunks/src/lib.rs

```diff
@@ -1356,23 +1393,27 @@ impl ShardsManager {
             partial_encoded_chunk.map(|chunk| PartialEncodedChunkV2::from(chunk));
         let header = &partial_encoded_chunk.header;
         let chunk_hash = header.chunk_hash();
-        debug!(
+        let _span = tracing::debug_span!(
             target: "chunks",
+            "process_partial_encoded_chunk",
             ?chunk_hash,
-            height = header.height_created(),
             shard_id = header.shard_id(),
+            height_created = header.height_created(),
+            height_included = header.height_included())
+        .entered();
+        tracing::debug!(
```

Comment:
Some calls are using qualified path and some an imported alias, despite of these calls being in the same function even. Would there be value in harmonizing this in either direction?

Reply:
Definitely, but I can't find strong arguments in favor of one or the other. Suggestions welcome.

Reply:
I personally approach writing code by using the qualified paths at first, but if I find myself repeating the same module more than two or three times in the same file, I’ll refactor to an `use`. In these files `use` is already present, so I suggest to take advantage of it.

---

Either option has its own cons and pros.

`tracing::debug!` in particular will work cratewide regardless of where the code is moved (or at least do so much more easily), it is also easier to just type out `tracing::debug!` a couple times than to think about whether `tracing::debug!` has been `use`d already in this specific module.

But in either case these benefits reduce the inconvenience of writing the code once, at the cost of the reader’s convenience (the text becomes ever-so-slightly more noisy). There’s an axiom about code being read many more times than it is written, too…